lib/Supervisor.js

```diff
@@ -216,10 +216,10 @@ function run(
     });
 
     server.on('listening', function () {
-      workers = _.range(0, processes).map(function (index) {
+      for (var i = 0; i < processes; i++) {
```

Comment:
I do like the functional style of the old approach. Do you think we could use something like this?

```js
workers = Array.from({length: process}).map((_, index) => {
```

Reply:
Why complicate it though?